wsd/FileServer.cpp

```diff
@@ -999,7 +1000,8 @@ bool FileServerRequestHandler::handleRequest(const HTTPRequest& request,
                                               requestUri.toString() + "].");
         }
 
-        if (endPoint == "welcome.html")
+        if (endPoint == "welcome.html" ||
+            endPoint == "debug.html")
```

Comment:
Are you sure we want to early exit here for debug.html?
We have more features like caching options or COOL_SERVE_FROM_FS below.

Also please note debug.html is not the thing used by integrators